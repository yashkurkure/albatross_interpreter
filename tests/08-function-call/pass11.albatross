var REf231MWgi7u0ImZbJ int := !45626&&31422&&39354^17552;
var k0waJ4wi int := REf231MWgi7u0ImZbJ<REf231MWgi7u0ImZbJ<REf231MWgi7u0ImZbJ<REf231MWgi7u0ImZbJ<=48291|34581^REf231MWgi7u0ImZbJ>REf231MWgi7u0ImZbJ>=REf231MWgi7u0ImZbJ||!REf231MWgi7u0ImZbJ>REf231MWgi7u0ImZbJ&REf231MWgi7u0ImZbJ|REf231MWgi7u0ImZbJ^REf231MWgi7u0ImZbJ;
var WqC9aIu6ldlQZ7Y86PPH string := "KgxH9VO0d6P2B4Bhez";
var gJKTQbFqNgh9VQJJscYb int := 43136-50798==21972||61623==25522+35278*k0waJ4wi&k0waJ4wi+REf231MWgi7u0ImZbJ;
var l4KphZjd4k1H0l9z string := "YgWzFfN01I7OzHngyM5";
fun AQlrTqK04SBbwj string () {l4KphZjd4k1H0l9z:=l4KphZjd4k1H0l9z;}
fun ahBygdx4 int () {if (!1309%5315<>43970>=35573-REf231MWgi7u0ImZbJ<=51073&1070&19862<=REf231MWgi7u0ImZbJ%REf231MWgi7u0ImZbJ%gJKTQbFqNgh9VQJJscYb&&15771-31881>=gJKTQbFqNgh9VQJJscYb==gJKTQbFqNgh9VQJJscYb) {while (gJKTQbFqNgh9VQJJscYb%35855<>gJKTQbFqNgh9VQJJscYb==REf231MWgi7u0ImZbJ<=7917|50652*36504<>gJKTQbFqNgh9VQJJscYb%k0waJ4wi+k0waJ4wi+31529%!gJKTQbFqNgh9VQJJscYb) {while (gJKTQbFqNgh9VQJJscYb<=REf231MWgi7u0ImZbJ|REf231MWgi7u0ImZbJ-gJKTQbFqNgh9VQJJscYb+63490&&39506-6846==19054==2839&24869&REf231MWgi7u0ImZbJ%!33904>!k0waJ4wi) {repeat (REf231MWgi7u0ImZbJ<>!REf231MWgi7u0ImZbJ>REf231MWgi7u0ImZbJ<=!12631*56503-gJKTQbFqNgh9VQJJscYb&&k0waJ4wi) {return !18883&30488>=44679<=gJKTQbFqNgh9VQJJscYb>gJKTQbFqNgh9VQJJscYb+gJKTQbFqNgh9VQJJscYb*11169||k0waJ4wi==REf231MWgi7u0ImZbJ^47152>REf231MWgi7u0ImZbJ<40875-gJKTQbFqNgh9VQJJscYb==!REf231MWgi7u0ImZbJ;}}}}}
fun fcJLquLhJQmO string () {WqC9aIu6ldlQZ7Y86PPH:="zG17rkJQfjVY";}
fun OXihidk4Oy void () {return;}
fun LkMX string () {l4KphZjd4k1H0l9z:="H0NaBHG8yGRtQykD";}
gJKTQbFqNgh9VQJJscYb:=15875*k0waJ4wi&&53706+19252>=31059-k0waJ4wi||40831&&44359>=REf231MWgi7u0ImZbJ|56770||739>gJKTQbFqNgh9VQJJscYb-!REf231MWgi7u0ImZbJ|53400||k0waJ4wi;
while (59747-42984>gJKTQbFqNgh9VQJJscYb*gJKTQbFqNgh9VQJJscYb+49283&26311|17818||!gJKTQbFqNgh9VQJJscYb&&gJKTQbFqNgh9VQJJscYb<>528%31891%k0waJ4wi>=gJKTQbFqNgh9VQJJscYb|35064) {repeat (REf231MWgi7u0ImZbJ-k0waJ4wi%36052==gJKTQbFqNgh9VQJJscYb-51838&REf231MWgi7u0ImZbJ<=gJKTQbFqNgh9VQJJscYb<=!!k0waJ4wi&&436==k0waJ4wi) {REf231MWgi7u0ImZbJ:=k0waJ4wi+REf231MWgi7u0ImZbJ&&k0waJ4wi==REf231MWgi7u0ImZbJ>1930||15742==24434<=gJKTQbFqNgh9VQJJscYb&50646==!27214<!53793<>!k0waJ4wi;}}
while (gJKTQbFqNgh9VQJJscYb<=k0waJ4wi==36340&63913==REf231MWgi7u0ImZbJ|gJKTQbFqNgh9VQJJscYb&&24867<4488<>REf231MWgi7u0ImZbJ<>gJKTQbFqNgh9VQJJscYb==52964<>k0waJ4wi&gJKTQbFqNgh9VQJJscYb|REf231MWgi7u0ImZbJ*30628) {return gJKTQbFqNgh9VQJJscYb||gJKTQbFqNgh9VQJJscYb*9547<>20740^54086&gJKTQbFqNgh9VQJJscYb<=k0waJ4wi<>k0waJ4wi&&1662<=14151*k0waJ4wi<=57027>=!58001||!REf231MWgi7u0ImZbJ;}
return gJKTQbFqNgh9VQJJscYb||k0waJ4wi||gJKTQbFqNgh9VQJJscYb&44338-gJKTQbFqNgh9VQJJscYb+gJKTQbFqNgh9VQJJscYb>51021&gJKTQbFqNgh9VQJJscYb>=48552&&!515<28608==gJKTQbFqNgh9VQJJscYb&&gJKTQbFqNgh9VQJJscYb||REf231MWgi7u0ImZbJ;
return 24625*k0waJ4wi*35079<=9937^36655>9568<63267&52215*64791;
