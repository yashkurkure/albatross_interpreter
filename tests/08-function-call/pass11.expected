Variable declared "REf231MWgi7u0ImZbJ" type int
Variable declared "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable declared "WqC9aIu6ldlQZ7Y86PPH" type string
Variable declared "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable declared "l4KphZjd4k1H0l9z" type string
Function declared "AQlrTqK04SBbwj" returns string
Variable read "l4KphZjd4k1H0l9z" type string
Variable written "l4KphZjd4k1H0l9z" type string
Function declared "ahBygdx4" returns int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "k0waJ4wi" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Function declared "fcJLquLhJQmO" returns string
Variable written "WqC9aIu6ldlQZ7Y86PPH" type string
Function declared "OXihidk4Oy" returns void
Function declared "LkMX" returns string
Variable written "l4KphZjd4k1H0l9z" type string
Variable read "k0waJ4wi" type int
Variable read "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "k0waJ4wi" type int
Variable written "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "k0waJ4wi" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "k0waJ4wi" type int
Variable read "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable written "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "k0waJ4wi" type int
Variable read "k0waJ4wi" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "k0waJ4wi" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "gJKTQbFqNgh9VQJJscYb" type int
Variable read "REf231MWgi7u0ImZbJ" type int
Variable read "k0waJ4wi" type int
